.vscode/launch/.env

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Adding this and many of the other files also doesn't give one a feeling of confidence about this pull request...